web/src/SidebarAccount.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from "react"
+import cookies from "js-cookie"
+import styled, { keyframes } from "styled-components"
+import {
+  Color,
+  Font,
+  FontSize,
+  SizeUnit,
+  AnimDuration,
+  ZIndex,
+} from "./style-helpers"
+import { ReactComponent as AccountIcon } from "./assets/svg/account.svg"
+import { ReactComponent as TiltCloudLogoSvg } from "./assets/svg/logo-Tilt-Cloud.svg"
+import ButtonLink from "./ButtonLink"
+import ButtonInput from "./ButtonInput"
+
+let SidebarAccountRoot = styled.div`
+  position: relative; // Anchor SidebarAccountMenu
+  width: 100%;
+`
+let SidebarAccountHeader = styled.header`
+  display: flex;
+  justify-content: flex-end;
+  padding-top: ${SizeUnit(0.5)};
+  padding-left: ${SizeUnit(0.5)};
+  padding-right: ${SizeUnit(0.5)};
+`
+let SidebarAccountLabel = styled.p`
+  font-family: ${Font.monospace};
+  font-size: ${FontSize.small};
+  color: ${Color.grayLightest};
+  margin-right: ${SizeUnit(0.5)};
+  opacity: 0;
+  transition: opacity ${AnimDuration.short} ease;
+  transition-delay: ${AnimDuration.default};
+`
+
+let SidebarAccountButton = styled.button`
+  display: flex;
+  align-items: center;
+  border: 0;
+  cursor: pointer;
+  background-color: transparent;
+  padding-left: ${SizeUnit(0.25)};
+  padding-right: ${SizeUnit(0.25)};
+  padding-top: ${SizeUnit(0.15)};
+  padding-bottom: ${SizeUnit(0.15)};
+
+  &:hover ${SidebarAccountLabel} {
+    opacity: 1;
+  }
+`
+let SidebarAccountIcon = styled(AccountIcon)`
+  fill: ${Color.blue};
+  transition: color ${AnimDuration.default} linear;
+
+  &:hover {
```

Comment:
i think you want this to be
${SidebarAccountButton}:hover &
to pin the hover effect to the button?